### PR TITLE
#167729770 Testing PT/Github Integration

### PR DESCRIPTION
**Pull request by okafor-chidimma**
#### What does this PR do?

Ensures that pivotal tracker and github integration was properly done

#### Description of Task to be completed?
When PRs are made, they automatically show up as comments/commit with their associated stories, giving more context to the story.

#### How should this be manually tested?
* Create a branch off the staging branch using the andela best practices naming convention
* Push the newly created branch to the github repo

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?

#167729770
#### Screenshots (if appropriate)
None
#### Questions:
None

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const app = express();
 
 app.use(cors());
 
-// Normal express config defaults
+// Normal express config defaults..
 app.use(require("morgan")("dev"));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
```
Comment (okabamac) [inline src/index.js:23]: Looks good to me

**Review by mthamayor — COMMENTED** [review]
Since this is just a test branch to check if the integration is successful(which it is), I don't think it's necessary to merge this pull request.

**Review by sebaztyn — APPROVED** [review]
Cool.

**Review by okabamac — APPROVED** [review]

**Comment by Stylll** [comment]
@okafor-chidimma Good job!. I'll be closing the PR since we've confirmed that the integration works.

**Pull request closed without merging**